setup.cfg

```diff
@@ -6,11 +6,13 @@ max-line-length = 110
 max-doc-length = 79
 ignore = E133, E203, E226, E228, N802, N803, N806, N812, N813, N815, N816, W503
 exclude =
-  python/lsst/ts/salobj/ddsutil.py,
```

Comment:
Is this exclusion correct? I wonder if it might be a merge problem. Can you please verify?

Reply:
The exclusion was correct at the time because ddsutil.py was from ADLink and did not pass flake8. We switched to the version of ddsutil.py in OpenSplice (once it was updated to fix a serious segfault) and got rid of the version in salobj. Thus there is no longer any need to exclude it from flake8 testing. it's gone!

Reply:
good! I was afraid I missed it when I did a rebase. Please, go ahead and merge this.